File: app/src/main/java/App/repositories/retrofit/responses/PersonaResponse.java
package App.repositories.retrofit.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import App.repositories.room.entities.Personas;

public class PersonaResponse {



    @SerializedName("status")
    @Expose
    private String status;

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("data")
    @Expose
    private Personas personas;

    public PersonaResponse(String status, String message, Personas personas) {
        this.status = status;
        this.message = message;
        this.personas = personas;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Personas getPersonas() {
        return personas;
    }

    public void setPersonas(Personas personas) {
        this.personas = personas;
    }
}

File: app/src/main/java/App/repositories/retrofit/responses/PersonaResponse.kt
package App.repositories.retrofit.responses

import App.repositories.room.entities.Personas
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PersonaResponse(@field:Expose @field:SerializedName("status") var status: String, @field:Expose @field:SerializedName("message") var message: String, @field:Expose @field:SerializedName("data") var personas: Personas)